<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>1050</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>91</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1050</width>
      <height>91</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(183, 184, 250, 252)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Sesión en curso</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>1050</width>
     <height>959</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="izq_cont">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>1050</width>
      <height>959</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="menu_cont">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <widget class="QPushButton" name="botonPregunta">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>380</y>
          <width>162</width>
          <height>156</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconos/recursos_formularios.qrc">
          <normaloff>:/iconos/preguntaicono.png</normaloff>:/iconos/preguntaicono.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="botonPremio">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>380</y>
          <width>162</width>
          <height>156</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconos/recursos_formularios.qrc">
          <normaloff>:/iconos/premioicono.png</normaloff>:/iconos/premioicono.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="botonSituacion">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>380</y>
          <width>162</width>
          <height>156</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconos/recursos_formularios.qrc">
          <normaloff>:/iconos/botonVolumen.png</normaloff>:/iconos/botonVolumen.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="botonEmociones">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>380</y>
          <width>162</width>
          <height>156</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconos/recursos_formularios.qrc">
          <normaloff>:/iconos/emocionesBoton.png</normaloff>:/iconos/emocionesBoton.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="botonPausar">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>640</y>
          <width>162</width>
          <height>156</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconos/recursos_formularios.qrc">
          <normaloff>:/iconos/boton-de-pausa.png</normaloff>:/iconos/boton-de-pausa.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>125</width>
          <height>125</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="botonReanudar">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>640</y>
          <width>162</width>
          <height>156</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconos/recursos_formularios.qrc">
          <normaloff>:/iconos/boton-de-play.png</normaloff>:/iconos/boton-de-play.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>125</width>
          <height>125</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>350</y>
          <width>119</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Contar Situación</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>350</y>
          <width>160</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Comprobar Emociones</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>350</y>
          <width>118</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Cozmo Pregunta</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>350</y>
          <width>90</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Recompensa</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>800</y>
          <width>69</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Reanudar</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>800</y>
          <width>50</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Pausar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="botonDetener">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>640</y>
          <width>162</width>
          <height>156</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconos/recursos_formularios.qrc">
          <normaloff>:/iconos/boton-detener.png</normaloff>:/iconos/boton-detener.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>125</width>
          <height>125</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>800</y>
          <width>59</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Detener</string>
        </property>
       </widget>
       <widget class="QSlider" name="barraVolumen">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>580</y>
          <width>481</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">Slider::groove:horizontal {
    border: 1px solid #999999;
    height: 8px;
    background: rgb(255, 255, 255);
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: rgb(0, 79, 172);
    border: 1px solid #5c5c5c;
    width: 18px;
    margin: -2px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
    border-radius: 3px;
}</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>590</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Volumen</string>
        </property>
       </widget>
       <widget class="QPushButton" name="botonEvaluacion">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>380</y>
          <width>162</width>
          <height>156</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconos/recursos_formularios.qrc">
          <normaloff>:/iconos/puntuaciones.png</normaloff>:/iconos/puntuaciones.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>135</width>
          <height>135</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>870</x>
          <y>350</y>
          <width>90</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Evaluación</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="camGraphicsView">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>90</y>
          <width>331</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>50</y>
          <width>90</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Vista cámara:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="botonCamara">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>640</y>
          <width>150</width>
          <height>150</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="iconos/recursos_formularios.qrc">
          <normaloff>:/iconos/botonCamara.png</normaloff>:/iconos/botonCamara.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>125</width>
          <height>125</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>800</y>
          <width>150</width>
          <height>15</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Iniciar o parar cámara</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="iconos/recursos_formularios.qrc"/>
 </resources>
 <connections/>
</ui>
